Makefile

```diff
@@ -5,6 +5,18 @@ all: clean build install-dev-dependencies up test
 build:
 	docker-compose build
 
+cf-crawl-save-staging: cf run-task dashboard-stage --command "php public/index.php campaign status omb-monitored download" --wait --name dashboard-omb-monitored-download
```

Comment:
If the commands are identical, why do we need different Makefile targets for the staging vs prod environments?

Reply:
This targets `dashboard-stage`, the application name is unique per environment because of new relic setup forcing app name to differ per space.

Reply:
Oh right, I forgot about that and missed the difference.